okio/build.gradle.kts

```diff
@@ -97,6 +97,9 @@ kotlin {
 
     val zlibTest by creating {
       dependsOn(commonTest)
+      dependencies {
+        implementation(libs.test.assertk)
```

Comment:
I will fix this wasm problem tomorrow (although I don't control when releases happen).

Reply:
I was playing with it. I ran into this:

```
* What went wrong:
Execution failed for task ':assertk:wasmWasiNodeTest'.
> command '/Users/jwilson/.gradle/nodejs/node-v21.0.0-v8-canary202309143a48826a08-darwin-arm64/bin/node' exited with errors (exit code: 1)
```

Totally unclear what I did wrong. Trying to borrow config from Okio wasn’t helping much!